minimisers/generalUtils/makeEmptyBayesResultsStruct.m

```diff
@@ -47,23 +47,23 @@
     
     reflectivityXData = cell(nContrasts,1);
     xDataCell = [1 1 1];
-    coder.varsize('xDataCell',[1e4 1e4],[1 1]);
+    coder.varsize('xDataCell',[1 1e4],[0 1]);
     for i = 1:nContrasts
         reflectivityXData{i} = xDataCell;
     end
     
     if isDomains
         sldXData = cell(nContrasts,2);
-        sldXDataCell = [1 1 1; 1 1 1];
-        coder.varsize('sldXDataCell',[2 1e4],[1 1]); 
+        domainSldXDataCell = [1 1 1; 1 1 1];
+        coder.varsize('domainSldXDataCell',[2 1e4],[0 1]); 
```

Comment:
I think this is meant to be the other way around

`coder.varsize('domainSldXDataCell',[2 1e4],[1 0]); `

Reply:
Are you sure? I've written that the fixed dimension has size 2 since there are always two domains, and the variable size dimension has size up to 1e4: https://uk.mathworks.com/help/coder/ref/coder.varsize.html